test/Macros/SwiftifyImport/CxxSpan/NoEscapeSpan.swift

```diff
@@ -0,0 +1,17 @@
+// REQUIRES: swift_swift_parser
+// REQUIRES: swift_feature_Span
+
+// RUN: %target-swift-frontend %s -swift-version 5 -module-name main -disable-availability-checking -typecheck -enable-experimental-feature Span  -plugin-path %swift-plugin-dir -dump-macro-expansions 2>&1 | %FileCheck --match-full-lines %s
+
+public struct SpanOfInt {
```

Comment:
Will the equivalent to this std::span representation be something that exists in the swift stdlib, or will it be generated by ClangImporter?

Reply:
This will be generated by the ClangImporter. I decided to mock this out to make the test more self contained.

Reply:
Ok, cool!